### PR TITLE
Add Google Sheets summary graphs to `studies.rst`

### LINKED ISSUE #3857: Display statistic graphs in the citing literature section
There is a way to embed googlesheet graphs as HTML's iframe, so I am wondering if we could do that in a sphinx-built page? If so, it would be neat to add a couple of visual statistics about the citing literature from [this source](https://docs.google.com/spreadsheets/d/1Utmf-btG7uXmu7RCBBx7fIKobm6yvXstX2Psc-UR2HA/edit#gid=1463629798).

Example of gsheet embedding [here](https://mcmedhacks.com/), with the HTML code for one example:

~~~
<iframe loading="lazy" width="600" height="371" seamless="" frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTMOgJ-8TyqT7IQGWDffe4krtXLgz8-9NO5d6HDr-coEHuvAlwOIfCjBl2fIpaP6iyWMQpyIk_lkJaz/pubchart?oid=1581143031&amp;format=interactive"></iframe>
~~~

### PR DESCRIPTION
**Pull request by joshuacwnewton**
## Description

SCT tracks citations in two ways: 

1. The public-facing [`studies.rst`](https://spinalcordtoolbox.com/overview/studies.html) page in our Sphinx documentation.
2. An internal, semi-private Google Sheets spreadsheet that tracks some additional information not in the Sphinx page.

The internal spreadsheet also contains some graphs that aggregate and summarize statistics for SCT's citations (such as pathologies treated, tools used, and type of subject). These graphs can be published and shared via Google Sheets' "publish" feature. 

This PR adds some HTML snippets to display the published graphs on the Sphinx page, alongside the list of citations. You can view the rendered graphs on this page: https://spinalcordtoolbox--3896.org.readthedocs.build/overview/studies.html

## Linked issues

Fixes #3857.

### CODE REVIEW

File: documentation/source/overview/studies.rst [hunk 1]
```diff
@@ -3,6 +3,24 @@
 Studies using SCT
```
Comment (joshuacwnewton) [inline documentation/source/overview/studies.rst:3]: Oops! Sorry, I've accidentally based my branch off of the `-stitch` PR (I forgot to switch to master before creating the branch).

Give me one moment and I'll rebase onto master.

Comment (joshuacwnewton) [reply documentation/source/overview/studies.rst:3]: There we are. :)

File: documentation/source/overview/studies.rst [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
 #################
 
+Summary graphs
+==============
+
+The graphs below summarize the pathologies, tools, species, and count per year of SCT citations:
+
+.. raw:: html
+
+       <iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSwyEvoiTOMflrJveD277xWYSb_1QSwkpxWsZoMSucgHBS7BHcgfvzGG21--1bLRFO_DIV4EhL9lBl2/pubchart?oid=1220039972&amp;format=interactive"></iframe>
```
Comment (joshuacwnewton) [inline documentation/source/overview/studies.rst:13]: I've added these snippets in the most straightforward way possible: The HTML formatting here is exactly what's provided by the Google Sheets UI for publishing. 

Still, I think it's worth discussing whether we're happy with the way this looks, or whether we want to try and make it look a different way via some HTML tweaks?

**Comment by jcohenadad** [comment]
@joshuacwnewton is it normal that i cannot see the graphs?

image

**Comment by joshuacwnewton** [comment]
My apologies! The link was incorrect. I've updated it now. (https://spinalcordtoolbox--3896.org.readthedocs.build/overview/studies.html)

**Review by jcohenadad — APPROVED** [review]
I ❤️ it!

**Comment by joshuacwnewton** [comment]
As this is a documentation pull request, I'm going to merge this PR even despite the GitHub Actions workflows not triggering, because the RTD has passed.

**Pull request merged**